docs/k8s-cluster/deep-learning-examples.md

```diff
@@ -0,0 +1,170 @@
+# NVIDIA Deep Learning Examples for Tensor Cores
+
+## Introduction
+
+This repository provides State-of-the-Art Deep Learning examples that are easy to train and deploy, achieving the best reproducible accuracy and performance with NVIDIA CUDA-X software stack running on NVIDIA Volta, Turing and Ampere GPUs.
+
+## NVIDIA GPU Cloud (NGC) Container Registry
+
+These examples, along with our NVIDIA deep learning software stack, are provided in a monthly updated Docker container on the NGC container registry (https://ngc.nvidia.com). These containers include:
+
+- The latest NVIDIA examples from this repository
+- The latest NVIDIA contributions shared upstream to the respective framework
+- The latest NVIDIA Deep Learning software libraries, such as cuDNN, NCCL, cuBLAS, etc. which have all been through a rigorous monthly quality assurance process to ensure that they provide the best possible performance
+- [Monthly release notes](https://docs.nvidia.com/deeplearning/dgx/index.html#nvidia-optimized-frameworks-release-notes) for each of the NVIDIA optimized containers
+
+## DeepOps Deployment Options
+
+Once you've set up a working cluster, and a [local docker registry](playbooks/k8s-cluster/container-registry.yml) accessible by all nodes, you may use the provided script to deploy a jupyter lab session on node port 30888.
```

Comment:
Link is broken because it's relative to `docs/k8s-cluster`. Please fix the link, and perhaps make it a little clearer that the local docker registry has to be configured before running any examples?